src/index.html

```diff
@@ -18,5 +22,85 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Company Logo"
```

Comment:
Remember about correct indentation between parent and child elements. The `img` tag should have a 2-space indentation from its parent `a` tag.